libminifi/include/utils/MinifiConcurrentQueue.h

```diff
@@ -21,20 +21,23 @@
 #include <deque>
 #include <mutex>
 #include <condition_variable>
+#include <utility>
 #include <stdexcept>
+#include <type_traits>
```

Comment:
`<type_traits>` is no longer used since `TryMoveCall` and `void_t` were extracted. Please fix headers to follow `include-what-you-use`

rationale:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf11-header-files-should-be-self-contained
If `utils/TryMoveCall.h` were not self-contained, following iwyu would more likely expose the issue in the form of compilation caure.

Reply:
Removed inclusion of `type_traits.h`.